### PR TITLE
Fixed prettier script

### PR DESCRIPTION
**Pull request by kurtisassad**
## Link to Issue
Closes: #5550

## Description of Changes
- Simplified prettier script making it run faster

## "How We Fixed It"
So according to the man page, the xargs -I creates a buffer to store, search and replace the string passed in. This buffer creation seems to be the cause of the issue because when I remove this option, the script runs as fast as before.

Although to fix this, I just removed the whole pipe chain completely, as prettier takes in a list of files anyways.

## Test Plan
- Run the prettier script on some files. Make sure that the time per file ~50ms

### CODE REVIEW

**Review by Rotorsoft — CHANGES_REQUESTED** [review]
The problem I see with this approach is that $FILES could be too long, and it will fail the hook

Can you try this?

```
#!/bin/bash

GRAY='\033[30m'  
RED='\033[31m' 
YELLOW='\033[33m'
NC='\033[0m' 

FILES=$(git diff --staged --name-only --diff-filter=d | sed 's| |\\ |g')

if [ -n "$FILES" ]
then
    echo -e -n $YELLOW
    echo -e "📐 prettier staged before commit..."
    echo "------------------------------------"
    echo -e $NC
    echo "$FILES" | xargs npx prettier --write --ignore-unknown 
    echo "$FILES" | xargs git add
    exit 0
fi
```

**Comment by kurtisassad** [comment]
Swapped out to rogers method as:
1. xargs without -I doesn't add too much overhead
2. It fails on windows if too many files are passed in (xargs breaks it up line by line, so it should never fail due to too many files)

**Review by Rotorsoft — APPROVED** [review]
🚀 

Using this option: [https://prettier.io/docs/en/precommit.html#option-5-shell-script]( https://prettier.io/docs/en/precommit.html#option-5-shell-script)

Once we decide to move eslint to pre-commit stage, we can use [lint-staged](https://prettier.io/docs/en/precommit.html#option-1-lint-stagedhttpsgithubcomokonetlint-staged)

File: scripts/pre-commit.sh [hunk 1]
```diff
@@ -5,16 +5,15 @@ RED='\033[31m'
 YELLOW='\033[33m'
 NC='\033[0m' 
 
-FILES=$(git diff --staged --name-only --diff-filter=d)
```
Comment (Rotorsoft) [inline scripts/pre-commit.sh:8]: why not keeping this comment for later?

**Comment by Rotorsoft** [comment]
hey @timolegros , can you approve this one? Thanks!

**Comment by timolegros** [comment]
> hey @timolegros , can you approve this one? Thanks!

Testing it out now will approve in a few minutes if it checks out.

**Review by timolegros — APPROVED** [review]
Confirmed that it ran faster but unable to properly recreate the original issue of merging master. Approving since this is an improvement in performance regardless.

**Pull request merged**